## Assembly
First read quality checked with FastQC.
 
 * How was quality assessed?
 

Becuase reads were pretrimmed, assembled PacBio(short reads) with Canu.
  * Why Canu?
  
  Canu [Koren and Walenz 2017] was branched from Celera Assembler in 2015, and specialized for single-molecule high-noise sequences. Efficient and accurate assembly of large repeats and closely related haplotypes are the challenges addressed with Canu, a complete reworking of Celera Assembler that is specifically designed for noisy single-molecule sequences. Canu can support nanopore sequencing, halves depth-of-coverage requirements, and improves assembly continuity while simultaneously reducing runtime by an order of magnitude on large genomes. These advances result from new overlapping and assembly algorithms, including an adaptive overlapping strategy based on tf-idf weighted MinHash and a sparse assembly graph construction that avoids collapsing diverged repeats and haplotypes. Celera is no longer maintained as of 2015, the original paper was likely in development when Celera was still the assembler of choice

Then assembled Illumina+Nanopore reads in Spades

  * Why Spades?
  
  Compare quality of assemblies to correct for biases. Assembly quality was assessed through Quast, by comparing the spades contigs to the canu contigs.

Check assembly quality by comparing contigs to NCBI ref genome generated by Zhang et. al.
-QUAST

Improve assembly(either from Spades or canu)
-Pilon



## References
1. Canu command: https://bpa-csiro-workshops.github.io/intro-ngs-manuals/modules/btp-module-denovo-canu/denovo_canu/
2. canu tutorial: https://canu.readthedocs.io/en/latest/tutorial.html
3. Canu quickstart: https://canu.readthedocs.io/en/latest/quick-start.html#quickstart
4. Spades manual: http://cab.spbu.ru/files/release3.12.0/manual.html
5. Quality checking fastqc: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
6. https://biology.stackexchange.com/questions/69560/how-to-pipe-bwa-mem-output-without-saving-sam-file

plot coverage and depth statistics of a bam file: https://www.biostars.org/p/104063/

extract all the mapped reads from bwa output: https://www.biostars.org/p/127992/

 Filter Mapped Reads With Samtools https://www.biostars.org/p/56246/

PacBio FASTQ reads to polish\correct a PacBio Fasta assembly without Quiver: https://www.biostars.org/p/258209/

Tools To Calculate Average Coverage For A Bam File?: https://www.biostars.org/p/5165/

Reads Per Contig From Sam File?: https://www.biostars.org/p/14246/
